documentation/using/importers/index.html

```diff
@@ -347,7 +347,7 @@ <h3 id="creating-a-new-scheduled-import">Creating a new scheduled import</h3>
 <p><img src="/images/importer-step3.png" alt="importer-step3"></p>
 <p>On first time creation the Job is automatically <code>Scanned</code> and <code>Imported</code>.</p>
 <h3 id="configure-scheduling-interval">Configure scheduling interval</h3>
-<p>The scheduling interval can be globally configured for all the Jobs. It is a global setting and not a per-Job one. This is achieved through the <code>services.update.interval</code> property in the <code>application.properties</code> configuration file that takes the value of <code>SERVICES_UPDATE_INTERVAL</code> environment variable. The value should be set to a valid <a href="https://en.wikipedia.org/wiki/Cron#CRON_expression">CRON expression</a> ; default is every 2 hours.</p>
+<p>The scheduling interval can be globally configured for all the Jobs. It is a global setting and not a per-Job one. This is achieved through the <code>services.update.interval</code> property in the <code>application.properties</code> configuration file that takes the value of <code>SERVICES_UPDATE_INTERVAL</code> environment variable. The value should be set to a valid <a href="https://docs.oracle.com/cd/E12058_01/doc/doc.1014/e12030/cron_expressions.htm">CRON expression in Spring format</a> ; default is every 2 hours.</p>
```

Comment:
I'm not a big fan of having a link to Oracle documentation... you were mentioning Spring documentation in your comment. I've found this link to be helpful: https://spring.io/blog/2020/11/10/new-in-spring-5-3-improved-cron-expressions
What do you think?